types/css-declaration-sorter/css-declaration-sorter-tests.ts

```diff
@@ -1,4 +1,4 @@
-import postcss = require('postcss');
+import postcss from 'postcss';
```

Comment:
Out of curiosity, why change this?

Reply:
`postcss` uses now `export default` to support both ESM and JS. This results in `This expression is not callable.` here.